bin/paragon-scripts.js

```diff
@@ -1,24 +1,33 @@
 #!/usr/bin/env node
 const chalk = require('chalk');
 const themeCommand = require('../lib/install-theme');
+const buildTokensCommand = require('../lib/build-tokens');
+const replaceVariablesCommand = require('../lib/replace-variables');
+const buildScssCommand = require('../lib/build-scss');
+const { sendTrackInfo } = require('../utils');
 
 // command: executor function
 const COMMANDS = {
   'install-theme': themeCommand,
+  'build-tokens': buildTokensCommand,
+  'replace-variables': replaceVariablesCommand,
+  'build-scss': buildScssCommand,
 };
 
 (async () => {
-  const [command] = process.argv.slice(2);
+  const [command, ...commandArgs] = process.argv.slice(2);
   const executor = COMMANDS[command];
 
+  sendTrackInfo(command, 'trackCLICommands');
```

Comment:
I would track only for specific command

Reply:
These functions will differ only in the transmitted ID, otherwise, the logic will remain unchanged. I think it's better to reuse the code and not duplicate it